tests/unit/forklift/test_legacy.py

```diff
@@ -2057,8 +2056,8 @@ def test_upload_fails_with_invalid_extension(self, pyramid_config,
 
         assert resp.status_code == 400
         assert resp.status == (
-            "400 Invalid file extension. PEP 527 requires one of: .egg, "
-            ".tar.gz, .whl, .zip (https://www.python.org/dev/peps/pep-0527/)."
+            "400 Invalid file extension: Use .egg, .tar.gz, .whl or .zip "
+            "extension. (PEP 527 https://www.python.org/dev/peps/pep-0527)"
```

Comment:
Probably can just lose the "PEP 527" here entirely

Reply:
Done.